diff --git a/man/local.mk b/man/local.mk
index 5f3f2152d..6736823d4 100644
--- a/man/local.mk
+++ b/man/local.mk
@@ -192,6 +192,7 @@ endif
 				$$argv$(EXEEXT))			\
 	&& : $${SOURCE_DATE_EPOCH=`cat $(srcdir)/.timestamp 2>/dev/null || :`} \
 	&& : $${TZ=UTC0} && export TZ					\
+	&& export LD_LIBRARY_PATH=$${OPENSSL_LD_LIBRARY_PATH} \
 	&& export SOURCE_DATE_EPOCH && $(run_help2man)			\
 		     --source='$(PACKAGE_STRING)'			\
 		     --include=$(srcdir)/man/$$name.x			\
